frontend/src/ScriptView.svelte

```diff
@@ -29,16 +10,6 @@
     justify-content: end;
   }
 
-  .buttonbar button {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    padding: 0.5em;
-    margin-left: 0.5em;
-    border: 0;
-    background: var(--main-bg-color);
-  }
```

Comment:
This code probably should not have been removed (or at least not all of it). It set up some important styles for the buttons in the script view that now no longer apply.

<img width="256" alt="image" src="https://github.com/redballoonsecurity/ofrak/assets/99368685/573e57fc-2b57-4158-a55d-62c0bbd86a5e">